venues/ICML.cc/2017/MLAV/python/submission_done.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
```

Comment:
I'll change it to create_invitiations.py to make it easier to find.
A line or two of comments at the top of scripts would make it a lot easier for me to figure out what they are used for. Should I be nervous when I see 'Not to spec' written at the top?